editor/editor_export_godot3.cpp

```diff
@@ -1980,7 +1981,190 @@ void EditorExportGodot3::_save_config(const String &p_path) {
+		if (res >= 0 && regexp.get_capture_count() == 3) {
+			line = regexp.get_capture(1) + "_physics_process(" + regexp.get_capture(2);
+		}
+		regexp.clear();
+
+		// Convert RawArray() => PoolByteArray()
+		regexp.compile("(.*)RawArray\\(\\)(.*)");
+		res = regexp.find(line);
+		if (res >= 0 && regexp.get_capture_count() == 3) {
+			line = regexp.get_capture(1) + "PoolByteArray()" + regexp.get_capture(2);
+		}
+		regexp.clear();
+
+		// Convert Vector2Array() => PoolVector2Array()
+		regexp.compile("(.*)Vector2Array\\(\\)(.*)");
+		res = regexp.find(line);
+		if (res >= 0 && regexp.get_capture_count() == 3) {
+			line = regexp.get_capture(1) + "PoolVector2Array()" + regexp.get_capture(2);
+		}
+		regexp.clear();
+
+		// Convert ReferenceFrame => Control
+		regexp.compile("(.*)ReferenceFrame(.*)");
+		res = regexp.find(line);
+		if (res >= 0 && regexp.get_capture_count() == 3) {
+			line = regexp.get_capture(1) + "Control" + regexp.get_capture(2);
+		}
+		regexp.clear();
+
+		// Try to detect a HTTPClient object
+		regexp.compile("[ \t]*([a-zA-Z0-9_]*)[ ]*=[ ]*HTTPClient\\.new\\(\\)");
+		res = regexp.find(line);
+		if (res >= 0 && regexp.get_capture_count() == 2) {
+			http_var = regexp.get_capture(1).strip_edges();
+		}
+		regexp.clear();
+
+		if (http_var != "") {
+			// Convert .connect( => .connect_to_host(
+			regexp.compile("(.*)" + http_var + "\\.connect\\((.*)");
+			res = regexp.find(line);
+			if (res >= 0 && regexp.get_capture_count() == 3) {
+				line = regexp.get_capture(1) + http_var + ".connect_to_host(" + regexp.get_capture(2);
+			}
+			regexp.clear();
+		}
+
+		int count;
+		int tries = 0;
+		do {
+			count = 0;
+
+			// Convert _pos( => _position(
+			regexp.compile("(.*)_pos\\((.*)");
+			res = regexp.find(line);
+			if (res >= 0 && regexp.get_capture_count() == 3) {
+				line = regexp.get_capture(1) + "_position(" + regexp.get_capture(2);
+				count++;
+			}
+			regexp.clear();
+
+			// Convert KEY_RETURN => KEY_ENTER
+			regexp.compile("(.*)KEY_RETURN(.*)");
+			res = regexp.find(line);
+			if (res >= 0 && regexp.get_capture_count() == 3) {
+				line = regexp.get_capture(1) + "KEY_ENTER" + regexp.get_capture(2);
+				count++;
+			}
+			regexp.clear();
+
+			// Convert .get_opacity() => .get_modulate().a
+			regexp.compile("(.*)\\.get_opacity\\(\\)(.*)");
+			res = regexp.find(line);
+			if (res >= 0 && regexp.get_capture_count() == 3) {
+				line = regexp.get_capture(1) + ".get_modulate().a" + regexp.get_capture(2);
+				count++;
+			}
+			regexp.clear();
+
+			// Convert .set_opacity(var) => .modulate.a = var
+			regexp.compile("([ \t]*)([a-zA-Z0-9_]*)[ ]*\\.set_opacity\\((.*)\\)(.*)");
+			res = regexp.find(line);
+			if (res >= 0 && regexp.get_capture_count() == 5) {
+				line = regexp.get_capture(1) + regexp.get_capture(2) + ".modulate.a = " + regexp.get_capture(3) + regexp.get_capture(4);
+				count++;
+			}
+			regexp.clear();
+
+			// Convert var.type == InputEvent.KEY => var.get_name() == "InputEventKey"
```

Comment:
This is not the recommended method for type checking.
Use `var is InputEventKey` instead.

Reply:
Just fixed it.